### ISSUE TITLE
Contextual menu selected items are not to spec

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#3242]
Are you willing to submit a PR to fix? (Yes)

Requested priority: (Normal) 

Products/sites affected: (if applicable)
OneDrive and SharePoint. Soon to be Outlook

The selected items currently have a grey background and blue check. There should be no background and neutral check.
Build:
![image](https://user-images.githubusercontent.com/20363037/32077287-bcdf7d5e-ba57-11e7-8202-9ee60f676c80.png)

Spec:
![image](https://user-images.githubusercontent.com/20363037/32077327-e16dba78-ba57-11e7-8f5d-728866d4725b.png)

**Comment by lynamemi** [comment]
@betrue-final-final do selected items have the same background colors for hover and focus states that are shown above for the unchecked items?

**Comment by betrue-final-final** [comment]
Yes selected and unselected use the same states.

**Comment by phkuo** [comment]
@betrue-final-final combo boxes have a similar visual paradigm, do the styles there change as well?

**Comment by betrue-final-final** [comment]
Combo box should be based on dropdown, so thought I would fix that first.

**Comment by phkuo** [comment]
Reopening to track fixing combo box styles as well.

**Comment by lynamemi** [comment]
Sorry, got distracted by bundle size issues and forgot to do this!

**Comment by suztocco** [comment]
Just curious when this may be fixed and updated in Fabric? I see this bug all over OWA and it looks pretty bad.

**Comment by phkuo** [comment]
@lynamemi fixed dropdowns already last week, but somebody still needs to fix combo boxes

**Comment by lynamemi** [comment]
I can take a look at making the same change to combo boxes today and tomorrow.

**Comment by suztocco** [comment]
@betrue-final-final @lynamemi - Just a comment that the main items on the context menus don't usually have a "selected" state bg color other than "checkmark" (except for if a submenu is selected). At least to my knowledge. Just want to make sure things don't get overwritten in certain areas that shouldn't be. Ben knows the specs but just wanted to call it out as I am not sure what all is being affected here.

**Comment by betrue-final-final** [comment]
Yes, that's what we're fixing.

**Comment by lynamemi** [comment]
Someone else has been assigned to the issue in #3252

**Comment by suztocco** [comment]
Just wanted to check on an ETA on this.

### We are launching our Outlook Calendar React Beta within the month and this component is used across the site.

adding @melvsparks (our dev) for visibility

**Comment by lynamemi** [comment]
I have been getting blocked by a couple code issues, but I am mostly there. I can go ahead and checkin what I have now and file a separate issue for the specific focus bug that is still outstanding.

**Comment by suztocco** [comment]
Thanks for replying.
Also, not sure if noted somewhere but **there is also a "press" state that is showing up in the control that isn't there in the design file**. The hover and press are the same color from what I see. 

![image](https://user-images.githubusercontent.com/20954039/32683271-f05ec634-c62e-11e7-9308-8128735983be.png)
@betrue-final-final  can clarify. I know if there is a submenu and it is selected that the bg is darker

**Comment by betrue-final-final** [comment]
Suz is correct about not having a pressed state. It should be same as hover. The origin of a submenu would be selected.

**Comment by suztocco** [comment]
@betrue-final-final  @lynamemi - Not sure if it was noted here but I have mentioned quite a few times that the submenu selected text is not semi-bold in the XD files but appears as so in code  
![image](https://user-images.githubusercontent.com/20954039/33045237-04cca580-ce01-11e7-844b-1eb22483025c.png)

![image](https://user-images.githubusercontent.com/20954039/33045261-17f1fcb4-ce01-11e7-8c52-b15ea5769bac.png)

**Comment by lynamemi** [comment]
I am looking into fixing this (the fix is straight forward), but I'm getting a weird scroll issue in my master branch that I need to figure out first. If someone else takes the font and background issue on, let me know.

**Comment by lynamemi** [comment]
Found the scroll bug! Someone replaced an example in a recent PR by mistake - should be able to get this fix in soon.

**Comment by lynamemi** [comment]
@suztocco , you can see the update if you follow the PR link referenced above.

**Comment by suztocco** [comment]
@lynamemi  I am looking around the page but not sure what/where to click.
In the past I could not SEE the actual design working either. It was just code.

**Comment by lynamemi** [comment]
I added screenshots to the description showing before and after. Unfortunately that's the best I can really get you from here. You would have to pull my code down onto your local computer to actually see it in action.

Alternatively, I can add gifs instead of static images if you want to see more of the experience.

**Comment by suztocco** [comment]
Ok thanks. I'm sure @betrue-final-final can test. He has the code system down. Looks like the bold is gone which was the item that felt the most off. Have a good holiday

**Comment by lynamemi** [comment]
Thanks, likewise!

I will go over it with Ben if I can catch him this afternoon.
